### DISCUSSION TITLE
Passwords do not appear and keep loading forever

### DISCUSSION THREAD

**Discussion by lionprodutosnaturais** [dani-garcia/vaultwarden#3747] [Q&A]
Hello, I'm new to this sector and I would like to ask for your help...

I installed Vaultwarden, version "vaultwarden/server:1.29.1-alpine", Docker-compose + Traefik.

Everything seems to be correct, from registration, confirming registration, browsing, creating an organization and everything else... But when I add a password it keeps loading forever and does not appear on the panel, also making it difficult to navigate the folders, I check the extension and the password was added, what could it be?

Docker-Compose:
`version: '3.9'

services:
  vaultwarden:
    container_name: vaultwarden
    image: vaultwarden/server:1.29.1-alpine
    restart: unless-stopped
    volumes:
      - vaultwardendb:/data/
    environment:
      LOGIN_RATELIMIT_MAX_BURST: '10'
      LOGIN_RATELIMIT_SECONDS: '60'
      ADMIN_RATELIMIT_MAX_BURST: '10'
      ADMIN_RATELIMIT_SECONDS: '60'
      ADMIN_TOKEN: 'xxxxxx'
      SENDS_ALLOWED: 'true'
      EMERGENCY_ACCESS_ALLOWED: 'true'
      WEB_VAULT_ENABLED: 'true'
      SIGNUPS_ALLOWED: 'false'
      SIGNUPS_VERIFY: 'true'
      SIGNUPS_VERIFY_RESEND_TIME: '3600'
      SIGNUPS_VERIFY_RESEND_LIMIT: '5'
      SMTP_HOST: 'xxxxxxxxxx'
      SMTP_FROM: 'xxxxxxxxxxxxxx'
      SMTP_FROM_NAME: 'xxxxxxxxxxx'
      SMTP_SECURITY: 'off'
      SMTP_PORT: '25'
      SMTP_USERNAME: 'xxxxxxxxxxxxx'
      SMTP_PASSWORD: 'xxxxxxxxxxxx'
      SMTP_AUTH_MECHANISM: 'Login'
    networks:
      - network_public
    deploy:
      mode: replicated
      replicas: 1
      placement:
        constraints:
        - node.role == manager
      labels:
      - traefik.enable=1
      - traefik.http.routers.vaultwarden.rule=Host(`xxxxxxxxxx.com.br`)
      - traefik.http.routers.vaultwarden.entrypoints=websecure
      - traefik.http.routers.vaultwarden.tls.certresolver=letsencryptresolver
      - traefik.http.routers.vaultwarden.service=vaultwarden
      - traefik.http.services.vaultwarden.loadbalancer.server.port=80

volumes:
  vaultwardendb:

networks:
  network_public:
    name: network_swarm_public
    external: true
...`

**Comment by lionprodutosnaturais** [comment]
Doing tests I verified that the problem is in the google chrome browser, because it opened in the mobile and edge browser and it worked normally, but I still haven't identified the possible problem... I opened it with other users and incognito window, and nothing...

**Reply by BlackDex** [reply]
Check the F12 developer console, that should probably give some indication on to why.

**Comment by lionprodutosnaturais** [comment]
Yes, I managed to find it, I don't know why, but the tiktok pixel extension was giving this problem, I removed it and dismissed it.
